recipes/confu_json/all/conanfile.py

```diff
@@ -28,39 +25,46 @@ def _minimum_cpp_standard(self):
     @property
     def _minimum_compilers_version(self):
         return {
-            "Visual Studio": "15",
+            "Visual Studio": "17",
             "gcc": "10",
             "clang": "10",
         }
 
-    def validate(self):
+
+       
+    def configure(self):
+        if self.settings.compiler == "Visual Studio" and Version(self.version)<Version("0.0.10"):
```

Comment:
```suggestion
        if is_msvc(self) and Version(self.version) < "0.0.10":
```
from conan.tools.microsoft import is_msvc
from conan.tools.scm import Version